components/notification-generator/tabs/sound/SoundTab.tsx

```
@@ -0,0 +1,383 @@
+        if (audio) {
+          audio.pause();
+          audio.currentTime = 0;
+        }
+        setPlaybackError(null);
+
+        const newAudio = new Audio(`${SOUND_BASE_URL}${sound.path}.ogg`);
+
+        let volumeValue = 1.0;
+        if (typeof notification.volume === "string") {
+          const parsedVolume = parseFloat(notification.volume);
+          if (!isNaN(parsedVolume) && isFinite(parsedVolume)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)